objectRecognitionSource/build.gradle

```diff
@@ -13,7 +13,16 @@ repositories {
     mavenCentral()
 
     maven {
-        url "http://vantiqmaven.s3.amazonaws.com/"
```

Comment:
Our pseudo-maven repo -- we don't have pom files or anything, just the underlying artifacts.  In gradle 6.something (I think), the resolver changed to use the pom files exclusively unless told otherwise.  The `artifacts()` statement here tells it to use the actual artifacts to resolve the dependency where appropriate.